File: Fundamentos de Programación/Ejercicio 2.11.py
import math

while True:
    try:
        r = float(input('Ingrese el radio de la circunferencia'))
        break
    except:
        continue

def circ(r):
    area = math.pi*r*r
    perimetro = math.pi*r*2
    return [area,perimetro]

[a,p] = circ(r)

print('El área del ciurculo es de ' + str(a) )
print('El perímetro del circulo es de ' + str(p))

exit()

File: Fundamentos de Programación/Ejercicio 2.11.cpp
#include <iostream>

int main()
{
    float r = 0, area = 0, perimetro = 0;
    double pi = 3.141592653589793;
    
    std::cout<<"Ingrese el radio de la circunferencia: \n";
    std::cin >>r;
    area = pi*r*r;
    perimetro = pi*r*2;
    std::cout<<"El área del circulo es de: "<<area;
    std::cout<<"El perímero del circuko es: "<<perimetro;
}